verification/certs.go

```diff
@@ -18,12 +123,12 @@ func getMultiTcbInfo(c *x509.Certificate) (TcgMultiTcbInfo, error) {
 	for _, ext := range c.Extensions {
 		if ext.Id.Equal(OidExtensionTcgDiceMultiTcbInfo) { // OID for Tcg Dice MultiTcbInfo
 			if !ext.Critical {
-				return multiTcbInfo, fmt.Errorf("[ERROR]: TCG DICE MultiTcbInfo extension is not marked as CRITICAL")
+				return multiTcbInfo, fmt.Errorf("multiTcbInfo extension is not marked as CRITICAL")
```

Comment:
Changed to smallcase as IDE kept complaining that error message should not start with Upper case,  have new lines and have punctuation at the end.